packages/webapp/src/containers/Navigation/index.jsx

```diff
@@ -40,6 +45,7 @@ const Navigation = ({ history, children, ...props }) => {
         resetSpotlight={resetSpotlight}
         history={history}
         isFarmSelected={isFarmSelected}
+        isAcceptingInvite={isAcceptingInvite}
```

Comment:
Should we just bake in this logic within the `hidden` prop instead of passing in another one? We could actually bake in `isFarmSelected` there too, and that way the child component doesn't care why it needs to show/hide, just that it does

Reply:
Hidden and showNavigation is separate. Hidden means navbar should not render at all (login) , showNavigation controls if any links or actions are visible in top nav and side nav (top menu with just logo).

I think seeing the logic defined here in the container is usefully explicit. The naming could maybe be improved to make that distinction?

Reply:
Ah, I think I was mis-reading the file above, gotcha! Yeah I think the naming is a bit confusing and could be improved, we could maybe go with something like `hideNav` and `hideTopMenuActions`, or the same but with `show`?

Reply:
Okay I went with showNav and showNavActions (since its not really just for top menu).